R/QCvalidate-helpers.R

```diff
@@ -182,7 +182,7 @@ qcidNietUitvoerbaar <- function(d, d_metingen, benodigdeKolommen){
     ungroup() %>% 
     select(qcid, iden)
   
-  niet_uitvoerbaar <- left_join(niet_uitvoerbaar, id, by = "iden")
+  niet_uitvoerbaar <- left_join(niet_uitvoerbaar, id)
```

Comment:
Ik vind het netter om bij een join explicitiet op te geven welke kolommen je voor de join gebruikt. Als je later de code aanpast kom je niet voor verassingen te staan.

Reply:
Dat had ik gedaan omdat zowel `niet_uitvoerbaar` als `id` ook de kolom qcid nodig hebben omdat de afkeuring niet gaat per monster, maar per stof (als ik het mij goed herinner). Alternatief kan ik qcid of stof toevoegen aan iden of ik kan qcid toevoegen aan by. Wat vind jij? Is het netter om verschillende versies van iden te hebben (in het gehele package) of is het netter om verschillende joins te doen?

Reply:
moet dit nog worden aangepast, of kan deze worden gemerged?

Reply:
Deze kan mi gemerged worden. De nieuwe dplyr versie test nu ook op many-to-many relations, en ik zie bij de joins nu veel warnings voorbij komen. Daar moet even gecontroleerd worden, ik heb daar #85 voor aangemaakt.